docs/contracts_api.md

```diff
@@ -0,0 +1,187 @@
+---
+id: contracts_api
+title: Contracts API
+---
+## Purpose
+
+The temporary housing relationship between various people requires an agreement in place to enbable the use of the property by Hackney, as a temporary accomodation asset. A landlord/hostel manager for example.
+
+As part of our solution, we are proposing to develop the Contracts API. This API will be designed to keep the asset and people models clean and provide a link between these objects as well as holding all the contractual information between the parties.
+
+## Our users and their needs
+
+** As a service user I need contract information so that: **
+
+- I can identify the responsible people at the property to contact.
+- I can identify the contract details such as rates, contract start and end dates.
+
+## Workshops
+
+This API was discussed at TDA 25/05/22 off the back of the need to move data off the asset data model.
+- https://docs.google.com/document/d/1MYJfzbop_yJhVGlSihhDmAhga6yrbxgCsZWujkYs8AA/edit?userstoinvite=lisa.stidle%40hackney.gov.uk#heading=h.xonwymseh0hm
+
+## Endpoints to be created
+
+### Get Contracts:
+
+**Endpoint Status**: PROPOSED
+
+**Purpose:**  
+Gets one or more Contracts from the Contract table.  Requests can be made by specifying a ref path parameter or searching for Tenures based on a series of parameters.
+
+**Endpoint URL's:**
+- /contracts
+- /contracts/?firstname
```

Comment:
Those will potentially be part of a 'search' endpoint, instead of the main API, otherwise we might need to use too make Dynamo GSIs